lib/pester/version.rb

```diff
@@ -1,3 +1,3 @@
 module Pester
-  VERSION = '0.1.0'
+  VERSION = '0.1.1'
```

Comment:
whatever you think we should do - i just tend to bump the minorest # until we hit 1.0